static/scripts/view.js

```diff
@@ -789,18 +789,160 @@ function checkFormValidity(event) {
   return valid;
 }
 
+
+var isWaitingOnAsyncCall = false;
+var hasShownSubmissionMessage = false;
+var isSubmittingChanges = false;
+
+function checkBugChange(bugid, lastMod, inMsg) {
+  var tiqitApi = new TiqitApi();
+  var msg = inMsg;
+
+  // Prepare the async event functions
+  bugChangeOnLoad = function(eventLoad) {
```

Comment:
Any reason to use an anonymous function? Since you're immediately giving it a name, this is (I think) equivalent to just defining `function bugChangeOnLoad(...)`. Not hugely important, but the latter is the "standard" way to define functions, unless there's a reason to do it differently.

Reply:
Mainly did it because the existing code did it elsewhere. I'll change it.